source/Plugins/Language/Swift/SwiftArray.cpp

```diff
@@ -68,7 +68,7 @@ SwiftArrayNativeBufferHandler::SwiftArrayNativeBufferHandler(
     : m_metadata_ptr(LLDB_INVALID_ADDRESS),
       m_reserved_word(LLDB_INVALID_ADDRESS), m_size(0), m_capacity(0),
       m_first_elem_ptr(LLDB_INVALID_ADDRESS), m_elem_type(elem_type),
-      m_element_size(0), m_element_stride(elem_type.GetByteStride()),
+      m_element_size(0), m_element_stride(0),
```

Comment:
I'm not sure about this. Above, in GetElementAtIndex, I see: `child_location = m_first_elem_ptr + idx * m_element_stride`

Perhaps it would be safer to remove the m_element_stride member and add a helper that returns Optional<size_t>. Or to store the stride as an Optional.

Reply:
That would be a functional change, which I tried explicitly to avoid in this first pass.
I agree with you though. Probably we should just error out if we can't find a stride? I can do that as a follow-up.

Reply:
I also think the same problem is true for size, which follows the same exact pattern.

Reply:
Sweet, no objections then.